sound/soc/sof/pcm.c

```diff
@@ -370,6 +363,9 @@ static int sof_pcm_trigger(struct snd_soc_component *component,
 		/* invoke platform trigger to stop DMA even if pcm_ops isn't set or if it failed */
 		if (!pcm_ops || !pcm_ops->platform_stop_during_hw_free)
 			snd_sof_pcm_platform_trigger(sdev, substream, cmd);
+		if (pcm_ops && pcm_ops->platform_stop_during_hw_free &&
```

Comment:
a comment and a blank line is needed to explain what we are doing?

Reply:
more comments are always better for this kind of code...